install_config/persistent_storage/topics/glusterfs_uninstall.adoc

```diff
@@ -8,15 +8,18 @@ ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansibl
 ----
 
 In addition, the playbook supports the use of a variable called
-`openshift_storage_glusterfs_wipe` which, when enabled, will destroy any data on
-the block devices that were used for {gluster} backend storage. It is recommend
-to use this variable like so:
+`openshift_storage_glusterfs_wipe` which, when enabled, destroys any data on the
+block devices that were used for {gluster} backend storage. To use the
+`openshift_storage_glusterfs_wipe` variable:
 
 ----
-ansible-playbook -i <path_to_inventory_file> -e "openshift_storage_glusterfs_wipe=true" /usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/uninstall.yml
+---- ansible-playbook -i <path_to_inventory_file> -e
 ----
+"openshift_storage_glusterfs_wipe=true"
+----
+/usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/uninstall.yml ----
```

Comment:
@tmorriso-rh I'm not sure what's happening above. There's been some extra ----'s added into the middle of the command? Looks like the command hasn't changed, so maybe just delete the extra lines that have been added.

You can see it better here:

https://raw.githubusercontent.com/tmorriso-rh/openshift-docs/0ffd0d4350d5a39e28fd366025db447feb7fb82f/install_config/persistent_storage/topics/glusterfs_uninstall.adoc

Also, like the other comment I made above, this command needs a prompt at the start.